pydantic/env_settings.py

```diff
@@ -20,8 +24,9 @@ class BaseSettings(BaseModel):
     Heroku and any 12 factor app design.
     """
 
-    def __init__(__pydantic_self__, **values: Any) -> None:
+    def __init__(__pydantic_self__, _env_file: Union[None, str, Path] = env_file_sentinel, **values: Any) -> None:
         # Uses something other than `self` the first arg to allow "self" as a settable attribute
+        object.__setattr__(__pydantic_self__, '_env_file', _env_file)
```

Comment:
instead of doing this, pass `_env_file` to `_build_values`

Reply:
you shouldn't. Please try to work this out yourself.

If you have trouble I'll take over the PR.

Reply:
Please remove this and simply pass `_env_file` to `_build_values` as a second argument, then to `_build_environ`